server.go

```diff
@@ -92,6 +93,14 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 		}(ret.Workers[i])
 	}
 
+	ret.EventWorker = NewEventWorker(ret.statsd)
```

Comment:
yeah, if we did have multiple of them we'd just distribute at random. i'm also not chunking the POST for this either. both of these things might get implemented once we know exactly how much volume we're generating.